Frends.Community.Multipart.SendMultipartRequest/SendMultipartRequest.cs

```diff
@@ -1,53 +1,86 @@
-
-#pragma warning disable 1591
-#pragma warning disable CS1573
-
-namespace Frends.Community.Multipart
-{
-    public static class MultipartTasks
-    {
-        /// <summary>
-        /// Send multipart/form-data request to server.
-        /// Documentation: https://github.com/CommunityHiQ/Frends.Community.Multipart.SendMultipartRequest
-        /// </summary>
-        /// <param name="input"/>
-        /// <param name="options"/>
-        ///  <returns>Object { Object Body, bool IsSuccessful, Exception ErrorException, string ErrorMessage }</returns>
+using System.IO;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+
+#pragma warning disable 1591
+#pragma warning disable CS1573
+
+namespace Frends.Community.Multipart
+{
+    public static class MultipartTasks
+    {
+        /// <summary>
+        /// Send multipart/form-data request to server.
+        /// Documentation: https://github.com/CommunityHiQ/Frends.Community.Multipart.SendMultipartRequest
+        /// </summary>
+        /// <param name="input"/>
+        /// <param name="options"/>
+        ///  <returns>Object { Object Body, bool IsSuccessful, Exception ErrorException, string ErrorMessage }</returns>
         public static async Task<SendResult> SendMultipartRequest(
             [PropertyTab] SendInput input,
             [PropertyTab] SendOptions options,
             CancellationToken cancellationToken)
         {
-            var client = new RestClient(input.Url);
-            var request = new RestRequest("/", Method.Post)
+            try
             {
-                AlwaysMultipartFormData = true
-            };
+                var secondsToTicks = (int)TimeSpan.FromSeconds(options.Timeout).Ticks;
+                var client = new RestClient(input.Url);
+                var request = new RestRequest("/", Method.Post)
+                {
+                    AlwaysMultipartFormData = true,
+                    Timeout = secondsToTicks
+                };
 
-            foreach (var file in input.FilePaths)
-            {
-                cancellationToken.ThrowIfCancellationRequested();
-                if (!File.Exists(file.Fullpath)) throw new FileNotFoundException("Input file was not found. File: " + file.Fullpath);
-                var filebyte = File.ReadAllBytes(file.Fullpath);
-                request.AddFile("file", filebyte, file.Name, "application/octet-stream");
-            }
+                foreach (var file in input.FilePaths)
+                {
+                    cancellationToken.ThrowIfCancellationRequested();
+
+                    if (!File.Exists(file.Fullpath)) 
+                        throw new FileNotFoundException("Input file was not found. File: " + file.Fullpath);
+
+                    var filebyte = File.ReadAllBytes(file.Fullpath);
+                    request.AddFile("file", filebyte, file.Name, "application/octet-stream");
+                }
+
+                foreach (var file in input.TextData)

```

Comment:
I think the variable `file` should be changed to something like `text`, since now it is a bit misleading since these are just string data.